recipes/r-ggrepel/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = '0.6.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ggrepel
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: ggrepel_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/ggrepel_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/ggrepel/ggrepel_{{ version }}.tar.gz
+  sha256: 360ae9d199755f9e260fefbd3baba3448fad3f024f20bcd9942a862b8c41a752
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rcpp
+    - r-ggplot2 >=2.0.0
+    - r-scales >=0.3.0
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - libgcc  # [not win]
+    - r-base
+    - r-rcpp
+    - r-ggplot2 >=2.0.0
+    - r-scales >=0.3.0{indent}libgcc  # [not win]
```

Comment:
The `{indent}` here and a few other recipes is causing the current error on Travis. If you update conda-build and then re-run `conda skeleton cran`, this will be fixed. As an example, here's what I get with conda-build 3.0.10:

```
  run:
    - r-base
    - r-rcpp
    - r-ggplot2 >=2.0.0
    - r-scales >=0.3.0
    - {{native}}gcc-libs   # [win]
    - libgcc               # [not win]
```